app/assets/stylesheets/styles.scss

```diff
@@ -49,16 +49,31 @@ h1 a:hover {
   margin-bottom: 0;
 }
 
+.release-image-show {
+  padding-right: 0;
+  margin-bottom: 20px;
+}
+
+.release-panel {
+  min-height: 470px;
+}
+
 .right-button {
-  margin-right: 20px;
   margin-top: -20px;
 }
 
+.right-button-release-show {
+  margin: -20px 0 20px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `-20px 0 20px` instead of `-20px 0 20px 0`